library/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScope.kt

```diff
@@ -42,7 +42,8 @@ internal class RumViewManagerScope(
     @WorkerThread
     override fun handleEvent(event: RumRawEvent, writer: DataWriter<Any>): RumScope {
         if (!applicationDisplayed) {
-            val isForegroundProcess = sdkCore.processImportance ==
+            val processImportance = sdkCore.getDatadogContext()?.processInfo?.processImportance
```

Comment:
I was actually thinking to remove process info from `DatadogContext`, because it seems it is something very specific and needed only in 1 place. Also if we are moving `processImportance` to the instance variable instead of global, can we have any issues popping up? Say we have 2 SDK instances, each got different values, depending on the time it was initialized. wdyt?

Reply:
Indeed, another option would be to keep this static but not in the CoreFeature, as it's only needed for RUM. What about having a ContentProvider in RUM that is initialized with the app's process, and reads the info on startup once, and make it available for everyone?